apps/pie-storybook/stories/pie-button.stories.tsx

```diff
@@ -141,6 +149,14 @@ const buttonStoryMeta: ButtonStoryMeta = {
             },
             if: { arg: 'type', eq: 'submit' },
         },
+        responsiveSize: {
+            description: 'Set the size of the button when set as responsive for wider viewports.',
+            control: 'select',
+            options: ['', ...responsiveSizes],
+            defaultValue: {
+                summary: 'productive',
+            },
```

Comment:
nit: can be addressed in a follow-up PR.

I think we can use an if statement to only render this if isResponsive is true and avoid adding an empty string to the options.

```suggestion
            if: { arg: 'isResponsive', eq: true },

```